_locales/nl/messages.json

```diff
@@ -0,0 +1,182 @@
+{
+  "extensionDescription": {
+    "message": "Minimalistische Keepass plugin gebaseerd op Web Extensions en KeepassHTTP",
+    "description": "The description for the extension, used in the manifest."
+  },
+  "fillUsernamePassDescription": {
+    "message": "Vul uw gebruikersnaam en paswoord vanuit het huidige actieve invoerveld",
```

Comment:
- see below for 'huidige actieve'
- I think '... invullen' would be better than 'Vul ... in'